packages/component-library/src/BaseMap/BaseMap.js

```diff
@@ -0,0 +1,82 @@
+import React, { PropTypes, Component } from 'react';
+import MapGL from 'react-map-gl';
+import { css } from 'emotion';
+import './mapbox-gl.css';
+
+const mapWrapper = css`
+  margin: auto;
+  max-width: 900px;
+`;
+
+export default class BaseMap extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      viewport:{
+        width: window.innerWidth > 900 ? 898 : window.innerWidth,
```

Comment:
May change this to work with responsive containers if it doesn't already

Reply:
I was meaning to change that. It's already inside a div and it's responsive, so the map resize when the window changes and stays centered. But I'll update it.